### ISSUE TITLE
Product stock filter not working after 3.6.X update

### ISSUE THREAD

**Issue by smallstar67** [woocommerce/woocommerce#23530]
**Product Stock Filter Not working**

Having this issue after woo commerce update to 3.6.1 and 3.6.2. The filter to sort through products based on stock status does not function.    Using Generate Press theme.

**Comment by claudiosanches** [comment]
Hello @smallstar67, can you please fill all the information:

<pre>
**Describe the bug**
A clear and concise description of what the bug is. Please be as descriptive as possible; issues lacking detail, or for any other reason than to report a bug, may be closed without action.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Expected behavior**
A clear and concise description of what you expected to happen.

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>
</pre>

Also only the "stock status" stopped to work? Have you tried with prices and more?

**Comment by michaeltieso** [comment]
Hiya. I was able to reproduce this as well on a couple different sites. I'll continue testing to see if there's anything in common.

**Comment by claudiosanches** [comment]
@michaeltieso This may happens when missing update the lookup table after update to 3.6, but after generating it should start work again.

**Comment by WebAssen** [comment]
Hi, i am having same issue, product stock filter not working when i choose to filter out the "SOLD items". How can i fix this? Thank you.

**Comment by smallstar67** [comment]
I'm not following.  How does one generate a look up table?

*--HK Bowman *



*"What is the difference between genius and stupidity? Genius has limits."
- Albert Einstein. *

**Comment by kloon** [comment]
This might be related to #23587, please test #23590 and let me know if that resolves the issue for you.

**Comment by NenadObradovic** [comment]
Hi guys,

I will confirm this issue on a couple sites. 

One way to resolve it is to go inside Dashboard panel -> WooCommerce -> Status -> Tools -> And clear **WooCommerce transients** and regenerate **Product lookup tables** (Only when both tools are clicked filter will show again).

Nenad

**Comment by kloon** [comment]
This can be related to #23587 or as @NenadObradovic mentioned incomplete DB upgrade, [see the wiki](https://github.com/woocommerce/woocommerce/wiki/3.6.x-notes-FAQ#i-am-having-issues-sortingsearching-by-sku-after-the-update)

**Comment by smallstar67** [comment]
Using the Woo Commerce tools and upgrading the database did not solve this issue at all.  These tasks were done before I started this thread and repeated with no difference.  Yoast SEO plugin is taking the blame but have not released a fix for this.

**Comment by claudiosanches** [comment]
@smallstar67 we released 3.6.3 today that fix an issue with Yoast SEO plugin.

**Comment by pitchou25** [comment]
I have have problem with filtring products, my woocoomerce version 3.7.1,
?orderby=title&order=asc
